packages/lib/constants/app.ts

```diff
@@ -6,3 +6,6 @@ export const APP_FOLDER = IS_APP_MARKETING ? 'marketing' : 'web';
 export const APP_BASE_URL = IS_APP_WEB
   ? process.env.NEXT_PUBLIC_WEBAPP_URL
   : process.env.NEXT_PUBLIC_MARKETING_URL;
+
+export const APP_DOCUMENT_UPLOAD_SIZE_LIMIT =
+  Number(process.env.NEXT_PUBLIC_DOCUMENT_SIZE_UPLOAD_LIMIT) ?? 50;
```

Comment:
I don't think this will work as expected

```
Number(undefined) ?? 50 // Becomes NaN
```

I think you meant something like this?

```
Number(undefined) || 50 // Becomes 50
```